e2e/cypress/integration/system_console/settings_spec.js

```diff
@@ -0,0 +1,241 @@
+            });
+
+        // # Save setting
+        cy.findByTestId('saveSetting').should('be.enabled').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Confirm that modal shows up
+        cy.get('#confirmModalLabel').should('be.visible').should('have.text', 'Confirm data retention policy');
+        cy.get('#confirmModalButton').should('be.enabled').click();
+
+        // # Change dropdown
+        cy.findByTestId('enableMessageDeletiondropdown').select('Keep all messages indefinitely');
+
+        // * Verify that button is disabled
+        cy.get('#adminConsoleWrapper .wrapper--fixed > .admin-console__wrapper').
+            within(() => {
+                cy.get('.job-table__panel button').should('be.disabled');
+            });
+
+        cy.findByTestId('saveSetting').should('be.enabled').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Confirm that modal shows up
+        cy.get('#confirmModalLabel').should('be.visible').should('have.text', 'Confirm data retention policy');
+        cy.get('#confirmModalButton').should('be.enabled').click();
+    });
+
+    it('MM-T1181: Compliance and Auditing: Run a report, it appears in the job table', () => {
+        cy.visit('/admin_console/compliance/monitoring');
+
+        // # Enable compliance reporting
+        cy.findByTestId('ComplianceSettings.Enabletrue').click();
+
+        cy.findByTestId('saveSetting').should('be.enabled').click().wait(TIMEOUTS.HALF_SEC);
+
+        // # Fill up the boxes
+        cy.get('#desc').clear().type('sample report');
+        const now = new Date();
+        cy.get('#to').clear().type(now.toLocaleDateString());
+        now.setDate(now.getDate() - 1);
+        cy.get('#from').clear().type(now.toLocaleDateString());
+
+        // # Run compliance reports
+        cy.get('#run-button').click().wait(TIMEOUTS.HALF_SEC);
+
+        cy.findByText('Reload Completed Compliance Reports').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Ensure that reports appear
+        cy.get('.compliance-panel__table tbody').children().should('have.length.greaterThan', 0);
+
+        // * Ensure that the report is correct
+        cy.get('.compliance-panel__table tbody tr').first().should('contain.text', 'Download');
+        cy.get('.compliance-panel__table tbody tr').first().should('contain.text', 'sample report');
+    });
+
+    it('MM-T1635: Channel listing is displayed correctly with proper team name', () => {
+        cy.visit('/admin_console/user_management/channels').wait(TIMEOUTS.FIVE_SEC);
+
+        // # Get the team name
+        cy.get('#channels .DataGrid .DataGrid_rows > :nth-child(1) > :nth-child(2)').
+            invoke('text').
+            then((name) => {
+                // # Click on the channel
+                cy.get('#channels .DataGrid .DataGrid_rows .DataGrid_row').first().click();
+
+                // * Confirm that the team name is same
+                cy.get('#channel_profile .channel-team').should('have.text', 'Team' + name);
```

Comment:
I'm getting failing test here. Not sure if valid bug or flaky test. Could you please check again?

Reply:
It's probably the wait timeout. Can you check if increasing timeout to 10 seconds on L206 make it pass?